### PR TITLE
PV2-1226: v1 datalock schema 

### PR DESCRIPTION
**Pull request by kuzinator**

### CODE REVIEW

File: src/SFA.DAS.Payments.Database.Legacy/DataLock/DataLockEventCommitmentVersions.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿CREATE TABLE [DataLock].[DataLockEventCommitmentVersions]
+(
+	DataLockEventId				uniqueidentifier			NOT NULL,
+	CommitmentVersion			bigint			NOT NULL,
```
Comment (carlbm) [inline src/SFA.DAS.Payments.Database.Legacy/DataLock/DataLockEventCommitmentVersions.sql]: This is varchar(25) on shadow v1

File: src/SFA.DAS.Payments.Database.Legacy/DataLock/DataLockEventPeriods.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿CREATE TABLE [DataLock].[DataLockEventPeriods]
+(
+	DataLockEventId			uniqueidentifier			NOT NULL,
+	CollectionPeriodName	varchar(8)		NOT NULL,
+	CollectionPeriodMonth	int				NOT NULL,
+	CollectionPeriodYear	int				NOT NULL,
+	CommitmentVersion		bigint			NOT NULL,
```
Comment (carlbm) [inline src/SFA.DAS.Payments.Database.Legacy/DataLock/DataLockEventPeriods.sql]: This is varchar(25) in shadow

File: src/SFA.DAS.Payments.Database.Legacy/DataLock/DataLockEventPeriods.sql [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+	IsPayable				bit				NOT NULL,
+	TransactionType			int				NOT NULL
+)
```
Comment (carlbm) [inline src/SFA.DAS.Payments.Database.Legacy/DataLock/DataLockEventPeriods.sql]: There's a 'TransactionTypesFlag' column missing. int/null
It's actually more important than TransactionType

File: src/SFA.DAS.Payments.Database.Legacy/DataLock/DataLockEvents.sql [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+(
+	Id							bigint				PRIMARY KEY IDENTITY(1,1),
+	DataLockEventId				uniqueidentifier	NOT NULL,
+	ProcessDateTime				datetime			NOT NULL,
+	IlrFileName					nvarchar(50)		NOT NULL,
+	SubmittedDateTime		    datetime			NOT NULL,
+	AcademicYear				varchar(4)    		NOT NULL,
+	UKPRN						bigint				NOT NULL,
+	ULN							bigint				NOT NULL,
+	LearnRefNumber				varchar(100)		NOT NULL,
```
Comment (carlbm) [inline src/SFA.DAS.Payments.Database.Legacy/DataLock/DataLockEvents.sql]: varchar(12) on shadow

File: src/SFA.DAS.Payments.Database.Legacy/DataLock/DataLockEvents.sql [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    AimSeqNumber				bigint				NOT NULL,
```
Comment (carlbm) [inline src/SFA.DAS.Payments.Database.Legacy/DataLock/DataLockEvents.sql]: int on shadow

File: src/SFA.DAS.Payments.Database.Legacy/DataLock/DataLockEvents.sql [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+	IlrStartDate				date				NULL,
+	IlrStandardCode				bigint				NULL,
+	IlrProgrammeType			int					NULL,
+	IlrFrameworkCode			int					NULL,
+	IlrPathwayCode				int					NULL,
+	IlrTrainingPrice			decimal(12,5)		NULL,
+	IlrEndpointAssessorPrice	decimal(12,5)		NULL,
+	IlrPriceEffectiveFromDate	date				NULL,
+	IlrPriceEffectiveToDate		date				NULL
+)
```
Comment (carlbm) [inline src/SFA.DAS.Payments.Database.Legacy/DataLock/DataLockEvents.sql]: Missing column `Status` int/not null

File: src/SFA.DAS.Payments.Database.Legacy/DataLock/DataLockEventCommitmentVersions.sql (hunk 1 above)
Comment (kuzinator) [reply src/SFA.DAS.Payments.Database.Legacy/DataLock/DataLockEventCommitmentVersions.sql]: updated

File: src/SFA.DAS.Payments.Database.Legacy/DataLock/DataLockEventPeriods.sql (hunk 2 above)
Comment (kuzinator) [reply src/SFA.DAS.Payments.Database.Legacy/DataLock/DataLockEventPeriods.sql]: updated

File: src/SFA.DAS.Payments.Database.Legacy/DataLock/DataLockEventPeriods.sql (hunk 3 above)
Comment (kuzinator) [reply src/SFA.DAS.Payments.Database.Legacy/DataLock/DataLockEventPeriods.sql]: added

File: src/SFA.DAS.Payments.Database.Legacy/DataLock/DataLockEvents.sql (hunk 4 above)
Comment (kuzinator) [reply src/SFA.DAS.Payments.Database.Legacy/DataLock/DataLockEvents.sql]: updated

File: src/SFA.DAS.Payments.Database.Legacy/DataLock/DataLockEvents.sql (hunk 5 above)
Comment (kuzinator) [reply src/SFA.DAS.Payments.Database.Legacy/DataLock/DataLockEvents.sql]: updated

File: src/SFA.DAS.Payments.Database.Legacy/DataLock/DataLockEvents.sql (hunk 6 above)
Comment (kuzinator) [reply src/SFA.DAS.Payments.Database.Legacy/DataLock/DataLockEvents.sql]: added

**Comment by kuzinator** [comment]
This has been approved and merged into DataLockEvents feature branch.

**Pull request closed without merging**